test/prepare-providers.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+config1="examples/standalone/providers.tf"
```

Comment:
Whats the usecase for this helper?

Reply:
The test/fixture/standalone requires a provider.tf to run. Since we already have a provider.tf into examples/standalone, during the test execution we'll hit in a loop and then in an error.

As workaround, I'm using this script to disable the provider.tf from examples/standalone when Go test is being executed.

Reply:
Gotcha, lets add this comment to the script